### ISSUE TITLE
build path issue

### ISSUE THREAD

**Issue by fehmi** [vuejs/vue-cli#330]
ı have to build like this. especialy when i use sass in a component.

vue build /Users/Name/data/root/vue/assets/src/js/main.js --watch --dist ../../Users/Name/Fehmi/data/root/vue/assets/dist/vue --prod --lib

funny but it works that way. could be better if we can write path only once. i don't realy get it maybe you can help.

thanks for the all good stuff

**Comment by egoist** [comment]
hmm can you be more specific? I don't get you either 😅

**Comment by fehmi** [comment]
i think there is a inconsistency between js folder's path and dist's path. i must point dist path relative to main.js path. it works that way when i don't use sass in the component file.

**Comment by egoist** [comment]
`dist` is relative to `process.cwd()`

**Comment by posva** [comment]
Closing due to inactivity. Please open a new issue with a reference to this one if you can follow up with more information.
